components/LinkWithSeason.tsx

```diff
@@ -3,9 +3,17 @@ import React from 'react';
 
 import { getQuerySeason } from '../utils/season';
 
+interface SeasonLinkProps extends LinkProps {
+  disabled?: boolean;
+}
+
 function LinkWithSeason(
-  props: React.PropsWithChildren<LinkProps>
+  props: React.PropsWithChildren<SeasonLinkProps>
```

Comment:
Didn't pick up this before, but it may be useful and clearer to destructure these props so we have `{ disabled, as, children, ...props }`

Reply:
It has been done